src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -117,7 +116,9 @@ public int Run(MergeOptions mergeOptions)
                     mergedLog.SchemaUri = mergedLog.Version.ConvertToSchemaUri();
 
                     FileSystem.DirectoryCreateDirectory(outputDirectory);
-                    outputFilePath = Path.Combine(outputDirectory, GetOutputFileName(_options, key));
+                    outputFilePath = Path.Combine(
+                        outputDirectory,
+                        GetOutputFileName(_options, key).ReplaceInvalidCharInFileName("."));
```

Comment:
>GetOutputFileName [](http://example.com/codeflow?start=24&length=17)

Why are you doing this replacement? It doesn't seem like a good idea that we would arbitrarily create a file name that the user didn't request. Why didn't you simply throw on receiving an argument with an invalid char? Is there a good reason why we're trying to manage this adverse condition?